File: 02 - Business Model Layer/StatisticalDataModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DbProject
{
    public class StatisticalDataModel
    {
        public string LongestWord { get; set; }
        public string ShortestWord { get; set; }
        public float WordsAvgLength { get; set; }
        public int HowMuchWordsInTheSong { get; set; }
        public List<string?> wordsAppearsTheMostInTheSong { get; set;}
        public List<int> howMuchWordsInVerses { get; set; }
        public List<int> howMuchWordsInSentences { get; set; }


        public StatisticalDataModel() {
        }


    }
}

File: Client/src/app/Models/statisticalData.model.ts
 export class StatisticalDataModel{
  
    constructor(public longestWord: string,
                public shortestWord: string,
                public wordsAvgLength: number,
                public howMuchWordsInTheSong: number,
                public wordsAppearsTheMostInTheSong: string[],
                public howMuchWordsInVerses: string[],
                public howMuchWordsInSentences: string[]
            )
            {}  
 }
